courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>* Add the version of the package you are using <code>google-cloud-automl==&lt;VERSION&gt;</code></p><p> </p><p>*  Install the package only if not already there. You can do it using the "||" or syntax in bash</p><pre class="ql-syntax" spellcheck="false">pip freeze | grep google-cloud-automl==&lt;VERSION&gt; || pip install google-cloud-automl==&lt;VERSION&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done</p>